### DISCUSSION TITLE
Real-Time Lyrics

### DISCUSSION THREAD

**Discussion by mooonns** [pear-devs/pear-desktop#2165] [Ideas]
Hello! I love this app but was wondering if somehow this extension could be implemented (https://chromewebstore.google.com/detail/better-lyrics-lyrics-for/effdbpeggelllpfkjppbokhmmiinhlmg) 
It's a real time lyrics extension and its pretty new but works really well and I'd love to use it in the app.

**Comment by AndhryTR** [comment]
![Cuplikan layar 2024-07-30 080024](https://github.com/user-attachments/assets/8272e7e6-64c2-411f-a076-7e7c8b676a8b)
I hope they can integrate it into this app.
